### DISCUSSION TITLE
See which nuclei barcodes contribute to each pseudobulk replicate?

### DISCUSSION THREAD

**Discussion by mfrenkel16** [GreenleafLab/ArchR#1929] [Questions / Documentation]
I'm curious if there's a way to see which nuclei barcodes are included in each pseudobulk replicate made by addGroupCoverages? I wanted to test out the recent footprinting paper from Buenrostro's lab (Hu et al. on biorxiv) and they require as input a list of which CBCs are part of each pseudobulk replicate grouping. Sorry if this is in the manual somewhere, but I wasn't able to find it.  

Thanks!

**Comment by rcorces** [comment]
use `returnGroups = TRUE` in `addGroupCoverages()`.
https://www.archrproject.com/reference/addGroupCoverages.html#:~:text=for%20parallel%20computing.-,returnGroups,but%20rather%20peaks%20are%20called%20from%20a%20TileMatrix%20(peakMethod%20%3D%20%22Tiles%22).,-parallelParam
